src/components/ProseMirrorInner.tsx

```diff
@@ -53,7 +53,9 @@ export function ProseMirrorInner({
   });
 
   const editorState =
-    "state" in editorProps ? editorProps.state : editorView?.state ?? null;
+    "defaultState" in editorProps
+      ? editorProps.defaultState
+      : editorProps.state;
```

Comment:
The fix is here: 
`<ProseMirror>` requires either `state` or `defaultState` to be passed in as props. If we choose one or the other the editorState value will never be `null`.